src/geo/map-view.js

```diff
@@ -154,20 +154,18 @@ var MapView = View.extend({
     this._unbindModel();
     this.map.bind('change:view_bounds_sw', this._changeBounds, this);
     this.map.bind('change:view_bounds_ne', this._changeBounds, this);
-    this.map.bind('change:zoom', this._setZoom, this);
```

Comment:
Because they have the same behaviour applying two times the center and the zoom, we just need one call for both attributes.